CHANGELOG.md

```diff
@@ -16,6 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- scriv-insert-here -->
 
+<a id='changelog-2.8.3'></a>
```

Comment:
Please read in the beginning of CHANGELOG.md how to work with changelog. You do not need to modify the file itself.